ports/opensubdiv/vcpkg.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "opensubdiv",
+  "version-string": "3.4.3",
+  "description": "An Open-Source subdivision surface library.",
+  "homepage": "https://github.com/PixarAnimationStudios/OpenSubdiv",
+  "supports": "windows & !arm & !uwp & !static"
```

```suggestion
  "supports": "windows & !arm & !uwp"
```
`static` means static library linkage, not CRT linkage. There is not currently a static CRT linkage tag, so you'll need to add those failures to the ci.baseline.txt file.